app/frontend/src/features/communities/CommunityPage/CommunityPage.tsx

```diff
@@ -177,79 +160,57 @@ export default function CommunityPage() {
   return (
     <div className={classes.root}>
       <HeaderImage community={community} className={classes.header} />
-      <div className={classes.topContainer}>
-        <div className={classes.topInfo}>
-          <Breadcrumbs aria-label="breadcrumb" className={classes.breadcrumbs}>
-            {community.parentsList
-              .map((parent) => parent.community)
-              .filter(
-                (
-                  communityParent
-                ): communityParent is CommunityParent.AsObject =>
-                  !!communityParent
-              )
-              .map((communityParent) => (
-                <Link
-                  to={routeToCommunity(
-                    communityParent.communityId,
-                    communityParent.slug
-                  )}
-                  key={`breadcrumb-${communityParent?.communityId}`}
-                >
-                  {communityParent.name}
-                </Link>
-              ))}
-          </Breadcrumbs>
-          <Typography variant="h1" className={classes.title}>
-            {COMMUNITY_HEADING(community.name)}
-          </Typography>
-          <Typography variant="body2" className={classes.description}>
-            {community.description} {MORE_TIPS}
-            <Link to="#"> here.</Link>
-          </Typography>
-        </div>
-        <div className={classes.navButtonContainer}>
-          <CircularIconButton id="findHostButton" label={FIND_HOST}>
-            <CouchIcon />
-          </CircularIconButton>
-          <CircularIconButton
-            id="eventButton"
-            label={EVENTS_LABEL}
-            linkTo={routeToCommunityEvents(
-              community.communityId,
-              community.slug
-            )}
-          >
-            <CalendarIcon />
-          </CircularIconButton>
-          <CircularIconButton id="localPointsButton" label={LOCAL_POINTS_LABEL}>
-            <LocationIcon />
-          </CircularIconButton>
-          <CircularIconButton
-            id="discussButton"
-            label={DISCUSSIONS_LABEL}
-            linkTo={routeToCommunityDiscussions(
-              community.communityId,
-              community.slug
-            )}
-          >
-            <EmailIcon />
-          </CircularIconButton>
-          <CircularIconButton
-            id="hangoutsButton"
-            label={HANGOUTS_LABEL}
-            disabled
-          >
-            <CouchIcon />
-          </CircularIconButton>
-        </div>
-      </div>
+      <div className={classes.body}>
+        <Hidden smDown>
+          <NavDesktop community={community} />
+        </Hidden>
```

Comment:
On a scale of 1-10, how terrible is this?

Reply:
@darrenvong @andreaowu

Reply:
I have mixed feels about that helper... but I think it's a balancing act. If this is the only thing that differs then I think it's ok.

Although if we start having them sprinkled all over our layout, then at that point we may as well have two versions which I believe we'd want to avoid right now if possible and try to make sure the layout work across all sizes as much as possible.